### ISSUE TITLE
Accessibility - Toggle does not work for keyboard

### ISSUE THREAD

**Issue by JasonHowell** [GSS-Cogs/dd-cms#873]
Feedback from user:

I just wanted to report an accessibility bug I saw on the climate and weather dashboards page. Currently the toggle between the visualisation view and the chart data does not work with keyboard navigation or assistive technology because it is implemented using a <div> element instead of a native interactive HTML element (e.g. a <button>). I can see this page is in beta and I'm sure this is an oversight, but I felt I ought to flag it.

When you tab through the page it skips the visualisation and chart data

![Screenshot 2023-03-29 at 15.34.38.png](https://images.zenhubusercontent.com/60e6ff7099f1d459ddcb95e0/b66cd5ae-d69d-47b5-96e2-d3e68950290e)

**Comment by JasonHowell** [comment]
Now needs a release.
